operations/mimir-tests/test-helm-parity.jsonnet

```diff
@@ -15,6 +15,7 @@ mimir {
     unregister_ingesters_on_shutdown: false,
     query_sharding_enabled: true,
     overrides_exporter_enabled: true,
+    etcd_replicas: 5,
```

Comment:
I'm afraid this file is the wrong for this test :(

`mimir-tests/test-helm-parity-generated.yaml` is used as a base file for testing Helm vs Jsonnet configuration. In Helm we only use 3 etc replicas.

Reply:
Perhaps we can make a copy of `test-defaults.jsonnet`, call it `test-etcd-replicas.jsonnet`, and generate yaml file for it? (Test in this case is just inspection that yaml file has 5 replicas specified).

Reply:
Yup, sure; let me have a stab on that :)

Reply:
@pstibrany PTAL.